package.json

```diff
@@ -176,5 +179,16 @@
 		"test-unit:watch": "npm run test-unit -- --watch",
 		"test-unit-php": "docker-compose run --rm wordpress_phpunit phpunit",
 		"test-unit-php-multisite": "docker-compose run -e WP_MULTISITE=1 --rm wordpress_phpunit phpunit"
+	},
+	"lint-staged": {
+		"packages/*/package.json": [
+			"wp-scripts lint-pkg-json"
```

Comment:
We should rather use `npm run lint-pkg-json` in here to ensure it always uses the universal command.

Reply:
@gziolo If we do that, then it will run for all of the package.json files and not just the ones that have changed.